### PR TITLE
Update readme to version 0.8.0

### PR DESCRIPTION
**Pull request by flora-hofmann-frequenz**

### CODE REVIEW

**Review by stefan-brus-frequenz — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,9 +44,9 @@ from frequenz.client.common.metric import Metric
 from frequenz.client.reporting import ReportingApiClient
 
 # Change server address if needed
-SERVICE_ADDRESS = "reporting.api.frequenz.com:443"
+SERVICE_URL = "grpc://reporting.api.frequenz.com:443?ssl=true"
 API_KEY = open('api_key.txt').read().strip()
-client = ReportingApiClient(service_address=SERVICE_ADDRESS, key=API_KEY)
+client = ReportingApiClient(service_url=SERVICE_URL, key=API_KEY)
```
Comment (cwasicki) [inline README.md:49]: This should be `server_url`

File: README.md [hunk 2]
```diff
@@ -44,9 +44,9 @@ from frequenz.client.common.metric import Metric
...
 ```
 
 Besides the microgrid_id, component_ids, and metrics, start, and end time,
```
Comment (cwasicki) [inline README.md:52]: We should also (have already) update the doc part on the CLI in L112.

File: README.md (hunk 1 above)
Comment (flora-hofmann-frequenz) [reply README.md:49]: Oops...

File: README.md (hunk 2 above)
Comment (flora-hofmann-frequenz) [reply README.md:52]: > doc part on the CLI in L112.
I am not quite sure what you mean by that? Can you clarify it pls? Then I will add it to the new PR.

Comment (cwasicki) [reply README.md:52]: This line https://github.com/frequenz-floss/frequenz-client-reporting-python/pull/95/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R112

Comment (flora-hofmann-frequenz) [reply README.md:52]: I will update this in a separate PR, as I also want to update `main` to the "new" default.

**Pull request merged**